### PR TITLE
Clean Stuff and remove outdated tests

### PR DESCRIPTION
**Pull request by aureliencousin**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "main": "lib/index.js",
   "scripts": {
     "test": "NODE_ENV=test JASMINE_CONFIG_PATH=jasmine.json istanbul cover --config .istanbul.yml jasmine",
-    "test:new": "jest --forceExit --detectOpenHandles",
+    "test:new": "jest --forceExit",
```
Comment (jBouyoud) [inline package.json]: You can also force --color options to have colors on CI ;-)

File: package.json [hunk 2]
```diff
@@ -75,7 +75,7 @@
   },
   "engines": {
     "node": ">=10.14",
-    "npm": ">=5.5.1"
+    "yarn": ">=1.10.1"
```
Comment (jBouyoud) [inline package.json]: "node": ">=10",
    "yarn": ">=1"

Comment (jBouyoud) [inline package.json]: Please commit yarn.lock file

File: package.json (hunk 1 above)
Comment (aureliencousin) [reply package.json]: done, in new separate commit

File: package.json (hunk 2 above)
Comment (aureliencousin) [reply package.json]: @jBouyoud yarn.lock was already versioned (actually yarn was already used on the project, it's just that the package.json was outdated...)

Comment (aureliencousin) [reply package.json]: @jBouyoud  fixed, amended both commits

**Pull request merged**